proto/forch_configuration.proto

```diff
@@ -89,7 +89,7 @@ message OrchestrationConfig {
   AuthConfig auth_config = 7;
 
   // orchestrated testing configurations
-  OrchestratedTestingConfig orch_testing_config = 11;
+  FOTConfig fot_config = 11;
```

Comment:
This should be FotConfig -- the reason is that in this context 'fot' is a word, so it maps cleanly to fot_config -- otherwise, following the rules of conversion FOTConfig would map to f_o_t_config

Reply:
Done